runconfig/parse.go

```diff
@@ -65,6 +65,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		flCpuShares       = cmd.Int64([]string{"c", "-cpu-shares"}, 0, "CPU shares (relative weight)")
 		flCpusetCpus      = cmd.String([]string{"#-cpuset", "-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
 		flCpusetMems      = cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
+		flBlkioWeight     = cmd.Int64([]string{"-blkio-weight"}, 0, "Block IO relative weight, range from 10 to 1000")
```

Comment:
Avoid using "range" here as a term; people may actually  try to set a *range* (e.g., `--blkio-weight=10-150`).

Perhaps;
"Block IO (relative weight), between 10 and 1000"

(I don't think the longer version can be used here, because the output would become too long for the help text)